timestamp-api/src/main/java/com/palantir/timestamp/TimestampRange.java

```diff
@@ -25,6 +25,7 @@
  *
  * @author bdorne
  */
+@SuppressWarnings("FinalClass") // Don't want to break the API
```

Comment:
Should probably state where the annotation came from, so `checkstyle:FinalClass`